community/caddy/APKBUILD

```diff
@@ -53,15 +54,22 @@ package() {
 		"$pkgdir"/etc/caddy \
 		"$pkgdir"/var/www
 
-	install -Dm755 "$srcdir"/$pkgname.initd \
-		"$pkgdir"/etc/init.d/$pkgname
 	install -Dm644 "$srcdir"/$pkgname.confd \
 		"$pkgdir"/etc/conf.d/$pkgname
 	install -Dm644 "$srcdir"/$pkgname.conf \
 		"$pkgdir"/etc/$pkgname/$pkgname.conf
 }
 
-sha512sums="b2d7ff08687c1b1331c018430e3458d4a5e4d5923803fabe563536ce372209ac0350efd4112a1099bfbfb517935d4f498613820d6b0685d5947a903227e15636  caddy-0.10.14.tar.gz
+_openrc() {
```

Comment:
There should be no need for this, just install the services from `package()` and use the [default `-openrc` subpackage function](https://git.alpinelinux.org/cgit/abuild/tree/abuild.in#n1723).

Reply:
fixed, thanks.